packages/Modal/README.md

```diff
@@ -0,0 +1,39 @@
+# Modal
+
+https://design.wegalvanize.com/p/components/modal
+
+## Components
+
+You can use any of the following components to compose the Modal:
+
+```jsx
+<Modal.Header />
```

Comment:
Shouldn't we have also `<Modal.Overlay />` like we do on the sidepanel? then:

<Modal>
  <Modal.Overlay {...customProps} />
</Modal>

Reply:
Also, `<Modal.FocusTrap>`?

Reply:
Added `<Modal.Overlay />`, but removed `<Modal.FocusTrap>`. Options for FocusTrap can be passed as:
```jsx
<Modal>
  ...
  <Modal.Overlay
    focusTrapOptions={{
      initialFocus: ...,
    }}
  />
  ...
</Modal>
```